core/src/test/java/org/apache/cxf/helpers/ServiceUtilsTest.java

```diff
@@ -173,4 +171,23 @@ private void setupSchemaValidationValue(Object value, boolean isRequestor) {
 
         control.replay();
     }
-}
\ No newline at end of file
+
+    @Test
+    public void testMakeNamespaceFromClassName() {
+        String nameSpace = ServiceUtils.makeNamespaceFromClassName("Foo", "http");
+        assertEquals("http://DefaultNamespace", nameSpace);
+
+        nameSpace = ServiceUtils.makeNamespaceFromClassName(this.getClass().getName(), "http");
```

Comment:
Better to split in 2 test cases I think

Reply:
@reta  Good idea since we have two different cases.  Done